src/app/side-nav/side-nav.component.html

```diff
@@ -0,0 +1,15 @@
+<mat-sidenav-container>
+    <mat-toolbar-row>
+      <button mat-flat-button (click)="opened=true" style="color: white; position: fixed">ChatBot<mat-icon>menu</mat-icon></button>
```

Comment:
better use a method for it 

`(click)="opened=true"`
`(click)="onMenuClick()"`

And do not assign values inside template